src/containers/reservations/Accordion.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState, useRef } from 'react';
+import { CaretDownOutlined, CaretUpOutlined } from '@ant-design/icons';
+import styled from 'styled-components';
+
+const AccordionButton = styled.button`
+  background-color: #61802e;
+  padding: 5px;
+  border: none;
+  outline: none;
+  transition: background-color 0.4s ease;
+`;
+
+const AccordionTextDiv = styled.div`
+  padding: 18px;
+`;
+
+const AccordionContentDiv = styled.div`
+  overflow: auto;
+  transition: height 0.4s ease;
+`;
+
+const AccordionSectionDiv = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+const Accordion: React.FC = ({ children }) => {
+  const [setActive, setActiveState] = useState(false);
+  const content = useRef(document.createElement('div'));
+
+  return (
+    <AccordionSectionDiv>
+      <AccordionButton onClick={() => setActiveState(!setActive)}>
```

Comment:
Pull this out to a function defined above, I need to add a lint rule for this - TLDR is lambdas in JSX bad performance. 

You should also reference prevState and assign the value to !prevState instead of referencing the current value directly. This prevents unexpected lifecycle issues.